### PR TITLE
Remove specific mappings for  upstream Maven Central deps

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: publish-to-maven-central/SDK4Mvn.aggr [hunk 1]
```diff
@@ -33,18 +33,16 @@
   <mavenMappings namePattern="(org\.eclipse\.jdt)\.core\.compiler\.batch" groupId="$1" artifactId="ecj" snapshot="false"/>
   <mavenMappings namePattern="(org\.eclipse\.jdt)(.*)" groupId="$1" artifactId="$1$2" snapshot="false"/>
   <mavenMappings namePattern="(org\.eclipse\.pde)(.*)" groupId="$1" artifactId="$1$2" snapshot="false"/>
-  <mavenMappings namePattern="(org\.eclipse)((?!(\.emf|\.jetty|\.ecf)).*)$" groupId="$1.platform" artifactId="$1$2" snapshot="false"/>
+  <mavenMappings namePattern="(org\.eclipse)((?!(\.emf|\.ecf)).*)$" groupId="$1.platform" artifactId="$1$2" snapshot="false"/>
```
Comment (merks) [inline publish-to-maven-central/SDK4Mvn.aggr:36]: If you remove jetty from here then jetty will map to the platform group and you will generate wrong dependencies for it:

image

**Review by merks — CHANGES_REQUESTED** [review]
It's good to use the available tool to check the impact of changes.

**Comment by mickaelistria** [comment]
I've fixed the faulty change

**Comment by merks** [comment]
BTW, I am looking at improving the analysis to find other such redundant mappings.  I think there is at least one more...

**Comment by mickaelistria** [comment]
>  I think there is at least one more...

Thanks for hinting, it's sshd; I'll submit yet another patch.

**Comment by mickaelistria** [comment]
> I am looking at improving the analysis to find other such redundant mappings

FWIW, given there are only a handful custom mappings left and it's relatively easy to verify whether they're still necessary by comparing with the .target file, I don't think it's worth investing too much in automating analysis; but your time you call ;)

**Comment by merks** [comment]
> > I think there is at least one more...
> 
> Thanks for hinting, it's sshd; I'll submit yet another patch.

That's the one!  😃

**Review by merks — APPROVED** [review]
Very good!

**Pull request merged**